README.md

```diff
@@ -1,83 +1,182 @@
+
+   This automatically updates the PR going from `draft` to `staging`.
+
+You've written a post!
+
+The editors will now review and edit the post. Please respond to any questions they ask or suggestions they make. Their aim is to make the post readable and useful to its target audience.
+
 
-### Contributing a third party blog post
+## Editors: editing and publishing a post
 
-If you would like to add a blog post that is actually a link to an existing third party blog post, you can follow the normal steps described above for creating a blog post. You simply need to add the following attributes to the liquid front matter: 
-- redirect_link: secure_url_to_3rd_party_post
-- permalink: /blog/redirected.html
+These steps are completed by the editors of the blog. They might ask questions or make suggestions to the author of the post. They might also make edits directly in the post to prepare it for publishing.
 
-Also provide a level 1 heading eg:
+1. Review the post on the [draft site](https://draft-openlibertyio.mybluemix.net/blog/) as linked from the issue.
 
-`= Creating a cool app with MicroProfile`
+   Ask the author to make changes by adding review comments to the PR.
 
-### Troubleshooting
+   For edits such as punctuation, formatting, highlighting, adding SEO details, or larger changes discussed with the author, the editor can make the edits directly in the author's branch and push the changes to `draft` branch, then rebuild the [draft site from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io) to check them.
+   
+   To check out the author's branch locally: `git fetch origin` then `git checkout -b branch_name origin/branch_name`, which creates a new local branch that's linked to their remote branch. When you've made changes, push them back to `origin/branch_name`.
+
+2. Add tags to the blog post:
+
+   a. In the author's branch, update the [blogs_tags.json](./blog_tags.json) file by adding the slug of the blog post (not including the date part of the file name) to the start of the `posts` array (1-2 entries per line) for each appropriate tag. Do this in an editor (such as VSCode) and make sure the syntax is correct.
+
+   b. Push the changes to `draft` branch as before and check that they get built correctly on the [draft site](https://draft-openlibertyio.mybluemix.net/blog/).
+
+2. When a publishing date has been decided:
+
+   * Check that the post looks fine.
+   
+   * Check that the author's details and the SEO details, including front matter, the title, and the filename slug, are appropriate for the post.
+   
+   * Check that the post has tags defined in the `blogs_tags.json` file in the same PR.
+
+   * If necessary, rename the file with the planned publication date.
+
+3. On the day of publication (or the day before):
```

Comment:
Given the recent update, we can have the posts go to staging and prod well ahead of the publication date since they will not be built on the prod site until at least that date.  Is this purposefully trying to limit the posts gong to prod to sameday or the day before intended publication?

Reply:
Yes, just trying to keep sight of what's going into staging and prod. Having the option to publish them earlier is useful but I'll leave the instructions like this for now and maybe update them when we've used it a bit more and know what happens in practice.